src/logistic_regression.py

```diff
@@ -1,34 +1,19 @@
-from sklearn.datasets import make_classification
-from sklearn.model_selection import train_test_split
-from sklearn.linear_model import LogisticRegression as lr_sklearn
-from concrete.ml.sklearn import LogisticRegression as lr_concrete
+import sys
+import atheris
+from concrete.ml.sklearn import LogisticRegression 
+from utils import initialize_models, mean_absolute_percentage_error, consume_bytes
 
-# Dataset to train
-x, y = make_classification(n_samples=100, class_sep=2, n_features=30, random_state=42)
+concrete_model, scikit_model, data_info = initialize_models(LogisticRegression)
 
-# Split the data-set into a train and test set,
-# each set is split into input and result.
-input_train, input_test, result_train, _ = train_test_split(
-    x, y, test_size=0.2, random_state=42
-)
+def compare_models(input_bytes):
+    data = consume_bytes(input_bytes, data_info, margin=0.1)
+    # Run the inference, encryption and decryption is done in the background
+    FHE_pred = concrete_model.predict(data, execute_in_fhe=True)
+    # Get scikit prediction
+    prediction = scikit_model.predict(data)
 
-# Start the concrete-ml logistic regression model
-concrete_model = lr_concrete()
-# Train
-concrete_model.fit(input_train, result_train)
+    # Get the mean percentage error to make sure the accuaracy is around 99%
+    assert(mean_absolute_percentage_error(prediction, FHE_pred) < 1)
```

Comment:
this throws `AssertionError: <EMPTY MESSAGE> `
```suggestion
    error = mean_absolute_percentage_error(prediction, FHE_pred)
    assert(error < 1) , f"Error: The prediction accuracy compared to scikit is less than 99%: the error percentage is {error}%"
```